Calculators/BMI_Calculator/readme.md

```diff
@@ -0,0 +1,3 @@
+# BMI Calculator
+## I have added the BMI calculator.
```

Comment:
you need to add the image here !
And you can just directly copy your image and paste it in readme file itself.. just like text copy and paste

Reply:
I have tried to copy and paste my image several times but it is not happening. So I have added the google drive link.